src/main/java/org/jitsi/jicofo/SSRCValidator.java

```diff
@@ -67,9 +72,9 @@
 
     /**
      * The limit sources count per media type allowed to be stored by
-     * the {@link Participant} at a time.
+     * each {@link Participant} at a time.
      */
-    private final int maxSourceCount;
+    private int maxSourceCount;
```

Comment:
why can't we set this earlier anymore? (and keep it final) i thought this was a preconfigured/hard-coded thing

Reply:
I added it when working on one of the tests, but when I'm looking at it now we can keep it final, will change that.

Reply:
I've just pushed a fix - will rebase and squash it after all reviews are done.